packages/backend/src/db/storage/templates-data.ts

```diff
@@ -0,0 +1,375 @@
+import { Template } from '@/graphql/__generated__/types.generated'
+
+const SEND_FOLLOW_UPS_ID = 'aae185f7-2592-4683-a812-2d412232e403'
+const SEND_A_COPY_OF_FORM_RESPONSE_ID = 'd06d1024-2f1d-46ca-b9b3-2d3d073eddfe'
+const SCHEDULE_REMINDERS_ID = '65e90f41-b605-4e83-bcd7-e4d2e349299d'
+const TRACK_FEEDBACK_ID = 'b237ff81-dbe9-4513-8135-0b59eec2de97'
+const ATTENDANCE_TAKING_ID = '04f95a37-46fe-455b-aa96-28c421379e1a'
+const UPDATE_MAILING_LISTS_ID = '8ec2728a-6e4a-49c7-8721-ef6d4eb1d946'
+const ROUTE_SUPPORT_ENQUIRIES_ID = '2a84e2f6-4806-46a2-890a-0dba1411b12f'
+const GET_LIVE_UPDATES_THROUGH_TELEGRAM_ID =
+  '9c964678-8e24-440e-b2fd-b42da4dea4b1'
+const SEND_MESSAGE_TO_A_SLACK_CHANNEL_ID =
+  'c88d03f7-862b-45e5-8a51-33e293236bd8'
+
+// Helper function to ensure templates cannot be modified
+function deepFreeze<T>(object: T): T {
+  if (Array.isArray(object)) {
+    object.forEach(deepFreeze)
+  } else {
+    // Freeze each property if it is an object
+    Object.getOwnPropertyNames(object).forEach((name) => {
+      const value = (object as any)[name]
+      if (value && typeof value === 'object') {
+        deepFreeze(value)
+      }
+    })
+  }
+  return Object.freeze(object)
+}
+
+/**
+ * Note that parameters will have {{Replace with __}}, these are
+ * placeholders for users to change them to the variable pills
+ */
+export const TEMPLATES: Template[] = deepFreeze<Template[]>([
+  // Template: Send follow ups
+  {
+    id: SEND_FOLLOW_UPS_ID,
+    name: 'Send follow ups',
+    description:
+      'Send follow up emails to respondents after they submit a form',
+    // Steps: formsg --> postman
+    steps: [
+      {
+        position: 1,
+        templateId: SEND_FOLLOW_UPS_ID,
+        appKey: 'formsg',
+        eventKey: 'newSubmission',
+        sampleUrl: 'https://form.gov.sg/66c2bb59c28ac9d16c1b62c9',
+      },
+      {
+        position: 2,
+        templateId: SEND_FOLLOW_UPS_ID,
+        appKey: 'postman',
+        eventKey: 'sendTransactionalEmail',
+        parameters: {
+          body: '<p style="margin: 0">Hi {{Replace with response 1 name}},</p><p style="margin: 0"></p><p style="margin: 0">We have received your registration for this event! More details will be sent to you nearer to the event date.</p><p style="margin: 0"></p><p style="margin: 0">Cheers,</p><p style="margin: 0">Event organising committee</p>',
+          subject: 'Thank you for registering!',
+          senderName: 'Event committee',
+          destinationEmail: '{{Replace with response 2 email}}',
```

Comment:
I'm not sure what this does yet, is it shown on frontend?

Reply:
stacey and i just wanted to represent "variables to change" as variable pills (so UI wise it is more striking and users should understand to change them) but they are not actually variables

Reply:
will do this in a later pr since `{{test}}` is now not a variable pill after the rich text editor fix